Signal/src/UIColor+OWS.h

```diff
@@ -14,6 +14,7 @@
 + (UIColor *)ows_darkGrayColor;
 + (UIColor *)ows_yellowColor;
 + (UIColor *)ows_reminderYellowColor;
++ (UIColor *)ows_reminderDarkYellowColor;
```

Comment:
See the screenshot.  White text doesn't show up well against the "reminder yellow" so I used a similar but darker yellow that WhatsApp uses for these kinds of purposes.